damage.py

```diff
@@ -14,67 +14,70 @@ def pre_mitigation_damage(
     All values regarding damage modifiers should include the buffs/debuffs coming from spells, summoner spells, or items
     from both the attacker AND the defender
     """
+    tot_off_stats = base_offensive_stats + bonus_offensive_stats
+    dmg_mod_perc_multiplier = 1 - damage_modifier_percent / 100
+
     if crit:
-        return (base_damage + bonus_damage) * damage_modifier_percent_mult_factor * (
-            1.75 + crit_damage
-        ) + damage_modifier_flat
-    return (base_damage + bonus_damage) * damage_modifier_percent_mult_factor + damage_modifier_flat
+        crit_multiplier = (1.75 + crit_damage)
+    else:
+        crit_multiplier = 1
+
+    return tot_off_stats * dmg_mod_perc_multiplier * crit_multiplier + damage_modifier_flat
```

Comment:
```suggestion
    return (tot_off_stats * crit_multiplier + damage_modifier_flat) * dmg_mod_perc_multiplier *
```
A revérifier mais normalement c'est comme ça